controllers/keystoneapi_controller.go

```diff
@@ -1015,6 +1189,21 @@ func (r *KeystoneAPIReconciler) generateServiceConfigMaps(
 		"TransportURL":     string(transportURLSecret.Data["transport_url"]),
 	}
 
+	// create httpd  vhost template parameters
+	httpdVhostConfig := map[string]interface{}{}
+	for _, endpt := range []service.Endpoint{service.EndpointInternal, service.EndpointPublic} {
+		endptConfig := map[string]interface{}{}
+		endptConfig["ServerName"] = fmt.Sprintf("keystone-%s.%s.svc", endpt.String(), instance.Namespace)
```

Comment:
keystone.ServiceName might be better than hard-coding 'keystone'

Reply:
done